### ISSUE TITLE
[Question] Ducking with liquidsoap ?

### ISSUE THREAD

**Issue by macbroadcast** [AzuraCast/AzuraCast#950]
hey folks, 
i wanna know a bit more about the liquidsoap configuration and if its possible to duck a radio stream with a second signal? Found this huge liquid soap config and wondering how to use a few of the controls
https://sourceforge.net/p/savonet/mailman/message/33302403/

Thanks

**Comment by LordHelmchen666** [comment]
Maybe you are searching for add? https://www.liquidsoap.info/doc-1.3.4/reference.html#add

> Mix sources, with optional normalization.

**Comment by macbroadcast** [comment]
Thanks for the great list of examples, but how can i use the commands in azuracast ? i am not sure if i can use the same terminology in the advanced playlists configuration.?

**Comment by LordHelmchen666** [comment]
You can use every valid code for liquidsoap. But not in playlist configuration. You have to put it in "Advanced Custom Configuration
This code will be inserted into your station's Liquidsoap configuration, below the playlist configuration and just before the Icecast output. Only use valid Liquidsoap code for this section!". Last point when you edit stations profile. For example i use `# Ausgabe normalisieren  
radio = nrj(radio)`

**Comment by macbroadcast** [comment]
Yep, found it , it's in the station configuration page and not in the playlist, my bad
![bildschirmfoto von 2018-11-17 15-36-02](https://user-images.githubusercontent.com/7961/48662176-90262d00-ea7e-11e8-80b3-7e5109245778.png) i am just afraid to break something :sweat_smile:
